app/src/main/java/ch/epfl/skysync/components/ConfirmFlightDetailUI.kt

```diff
@@ -0,0 +1,270 @@
+package ch.epfl.skysync.components
+
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.PaddingValues
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxHeight
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.rememberScrollState
+import androidx.compose.foundation.verticalScroll
+import androidx.compose.material3.Button
+import androidx.compose.material3.ButtonDefaults
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.OutlinedTextField
+import androidx.compose.material3.Text
+import androidx.compose.material3.TextFieldDefaults
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.platform.testTag
+import androidx.compose.ui.text.style.TextOverflow
+import androidx.compose.ui.unit.dp
+import ch.epfl.skysync.models.flight.ConfirmedFlight
+import ch.epfl.skysync.models.flight.RoleType
+import ch.epfl.skysync.models.flight.Team
+import ch.epfl.skysync.models.flight.Vehicle
+import ch.epfl.skysync.ui.theme.lightOrange
+
+/**
+ * Composable function for displaying a UI to confirm flight details.
+ *
+ * @param confirmedFlight The details of the confirmed flight.
+ * @param backClick Lambda function to handle the click event for going back.
+ * @param paddingValues The padding values to be applied to the layout.
+ * @param confirmClick Lambda function to handle the click event for confirming the flight.
+ */
+@Composable
+fun ConfirmFlightDetail(
+    confirmedFlight: ConfirmedFlight,
+    backClick: () -> Unit,
+    paddingValues: PaddingValues,
+    confirmClick: () -> Unit
+) {
+  Column(modifier = Modifier.fillMaxHeight().fillMaxWidth().padding(paddingValues)) {
+    Column(modifier = Modifier.fillMaxHeight(0.87f)) {
+      Header(backClick = backClick, title = "Confirm Flight")
```

Comment:
**Confirmed Flight** instead of **Confirm Flight** might be less confusing since we already have a Confirm Flight screen